p2p.go

```diff
@@ -66,6 +65,8 @@ type (
 		EnableRateLimit          bool            `yaml:"enableRateLimit"`
 		RateLimit                RateLimitConfig `yaml:"rateLimit"`
 		PrivateNetworkPSK        string          `yaml:"privateNetworkPSK"`
+		GroupID                  string          `yaml:"groupID"`
+		ServerMode               bool            `yaml:"serverMode"`
```

Comment:
1. remove `ServerMode`? we discussed it needs to be server mode, so no need to change regarding this
2. add a blacklist timeout value, see comment below